src/thunderbird_accounts/asgi.py

```diff
@@ -24,6 +24,7 @@ def immutable_file_test(path, url):
 
 
 application = get_asgi_application()
-application = ServeStaticASGI(
-    application, root=settings.STATIC_ROOT, prefix='static', immutable_file_test=immutable_file_test
-)
+if not settings.DEBUG:
```

Comment:
For non-local dev envs we let staticserve serve the files that were collectstatic'd on.